css/src/tokens/typography.scss

```diff
@@ -9,3 +9,11 @@ $font-size-3: 1.75rem !default; // 28px
 $font-size-2: 2.125rem !default; // 34px
 $font-size-1: 2.5rem !default; // 40px
 $font-size-0: 3.375rem !default; // 54px
+
+// Font Weight
+$weight-normal: 400;
```

Comment:
@v-mingho We can add the variables for all the weight here. (We no longer need to use the $dont-use prefix either)